x-pack/plugins/enterprise_search/public/applications/workplace_search/views/groups/components/groups_table.tsx

```diff
@@ -79,7 +77,7 @@ export const GroupsTable: React.FC<{}> = () => {
         <EuiTableHeader>
           <EuiTableHeaderCell>{GROUP_TABLE_HEADER}</EuiTableHeaderCell>
           <EuiTableHeaderCell>{SOURCES_TABLE_HEADER}</EuiTableHeaderCell>
-          {!isFederatedAuth && <EuiTableHeaderCell>{USERS_TABLE_HEADER}</EuiTableHeaderCell>}
+          {<EuiTableHeaderCell>{USERS_TABLE_HEADER}</EuiTableHeaderCell>}
```

Comment:
No need for the wrapping `{}` braces anymore
```suggestion
          <EuiTableHeaderCell>{USERS_TABLE_HEADER}</EuiTableHeaderCell>
```